components/validator/WebApp/validator/templates/master.kid

```diff
@@ -30,7 +30,7 @@
         <div id="problem" style=>
             <h2>Problem with validator</h2>
             <p>We are experiencing a problem with the online validator hanging while validating files. This appears to be a problem related to the combination of XML parser and web application toolkit we are using. Unfortunately we cannot get this to work reliably so will have to completely re-write the validator. If you need to validate files we recommend to do so using the Bio-Formats command line <code>bftools</code>.</p>
-            <p>For further information check the <a href="http://www.openmicroscopy.org/site/support/faq/ome-xml-and-ome-tiff/how-do-i-validate-a-file-on-the-command-line">OME FAQ</a>. </p>
+            <p>For further information check the <a href="https://www.openmicroscopy.org/site/support/ome-model/ome-tiff/tools.html#validating-ome-xml">OME Model and Formats Documentation</a>. </p>

```

Comment:
That was me pasting the link in from my browser while logged in, I'll change it to http